File: Day 5/averageHeight.py
# Input a Python list of student heights
student_heights = input().split()
for n in range(0, len(student_heights)):
  student_heights[n] = int(student_heights[n])
# 🚨 Don't change the code above 👆
  
# Write your code below this row 👇

number_of_students = 0
total_height = 0


for i in student_heights:
  total_height += i
  number_of_students += 1

average_height = round(total_height / number_of_students)

print(f'total height = {total_height}')
print(f'number of students = {number_of_students}')
print(f'average height = {average_height}')

File: Day 5/javascript/averageHeight.js
// Input a JavaScript list of student heights
let student_heights = [180, 124, 165, 173, 189, 169, 146];


let number_of_students = 0;
let total_height = 0;

for (let i = 0; i < student_heights.length; i++) {
    total_height += student_heights[i];
    number_of_students += 1;
    
};
// console.log(total_height) 
// console.log(number_of_students)

let average_height = Math.round(total_height / number_of_students);

console.log('total height = ' + total_height);
console.log('number of students = ' + number_of_students);
console.log('average height = ' + average_height);